### PR TITLE
[ADD] base_location_nuts_no_update

### PR DESCRIPTION
**Pull request by SimoRubi**
This modification allows to mark some NUTS regions as not updatable.

### CODE REVIEW

File: base_location_nuts_no_update/README.rst
Comment (tarteo) [inline base_location_nuts_no_update/README.rst]: Can you use the seperated readme to generate the README, please?

File: base_location_nuts_no_update/README.rst
Comment (tarteo) [inline base_location_nuts_no_update/README.rst]: Can you add more information why updating some regions is desirable?

**Review by tarteo — CHANGES_REQUESTED** [review]

File: base_location_nuts_no_update/README.rst
Comment (SimoRubi) [reply base_location_nuts_no_update/README.rst]: Hi @tarteo, thanks for your review. 
Sorry but I don't get what you are suggesting here, I already have generated this `README.rst` using [this script](https://github.com/OCA/maintainer-tools/blob/master/tools/gen_addon_readme.py)

File: base_location_nuts_no_update/README.rst
Comment (SimoRubi) [reply base_location_nuts_no_update/README.rst]: Done in https://github.com/OCA/partner-contact/pull/628/commits/ea56223b60afb4865e9e9638b317c1b046b914b1

File: base_location_nuts_no_update/README.rst
Comment (tarteo) [reply base_location_nuts_no_update/README.rst]: Oh yes, I see :stuck_out_tongue:. Oops

**Review by tarteo — APPROVED** [review]
Thanks!

**Comment by SimoRubi** [comment]
@tarteo I changed the code to include the changes in the base module, I included what you suggested in your previous review, but you may want to review that again.

File: base_location_nuts/wizard/nuts_import.py
Comment (tarteo) [inline base_location_nuts/wizard/nuts_import.py]: You can do ```lambda n: not n.not_updatable``` but it does the same so it's not really important.

**Review by tarteo — APPROVED** [review]
I think it's also nicer in the base module. Code still looks good just a non-blocking comment.

File: base_location_nuts/wizard/nuts_import.py
Comment (SimoRubi) [reply base_location_nuts/wizard/nuts_import.py]: yes @tarteo but I wanted to avoid the double negation

**Comment by pedrobaeza** [comment]
Isn't better to integrate this on main module?

**Comment by SimoRubi** [comment]
@pedrobaeza yes during the PR I included that in the main module, I just noticed the description is still the old one; now it is correct

File: base_location_nuts/wizard/nuts_import.py
Comment (pedrobaeza) [reply base_location_nuts/wizard/nuts_import.py]: I prefer the other option too. Double negation is correct, and more in programming

Comment (SimoRubi) [reply base_location_nuts/wizard/nuts_import.py]: ok this is a plebiscite then, fixed that in https://github.com/OCA/partner-contact/pull/628/commits/1d555b8d43169161d4f3f80c0e770894dbdb87e9

**Review by pedrobaeza — APPROVED** [review]

**Review by Sashimi22 — APPROVED** [review]
Test functional OK

**Comment by pedrobaeza** [comment]
Please squash fixup before merging

**Comment by SimoRubi** [comment]
@pedrobaeza fixed everything up into one commit

**Comment by pedrobaeza** [comment]
Well, the 2 of them were fine, to separate the split of the README (you had 3), but no problem this way also.

**Comment by pedrobaeza** [comment]
Can you forward-port it to v11?

**Comment by SimoRubi** [comment]
ok I'm on it

**Pull request merged**
